clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -87,7 +87,7 @@ export const CustomSelect = ({
               display: "none",
             }),
           }}
-          isSearchable={isSearchable ?? false}
+          isSearchable={isSearchable}
```

Comment:
My only change was to remove the unnecessary fallback, inspired by the change on L151